latest/schema.json

```diff
@@ -356,13 +356,6 @@
         }
     },
     "properties": {
-        "api-version": {
```

Comment:
Let's make sure we add info saying that this property is deprecated. Instead of removing. Otherwise we will start seeing this error: `Property api-version is not allowed.yaml-schema`.